BenchmarkVCFs/FindSamplesAndBenchmark.wdl

```diff
@@ -36,17 +37,25 @@ workflow FindSamplesAndBenchmark {
         # Input for monitoring_script can be found here: https://github.com/broadinstitute/palantir-workflows/blob/main/Scripts/monitoring/cromwell_monitoring_script.sh.
         # It must be copied to a google bucket and then the google bucket path can be used as the input for monitoring_script.
         File monitoring_script
+
+        String? dummyInputForTerraCallCaching
+    }
+
+    parameter_meta {
+      input_callset_labels: {description:"Labels for each input callset that is being evaluated. This will include this label in the output summary table under the 'Name' column."}
     }
 
     Int VCF_disk_size = ceil(size(input_callset, "GiB")) + 10
 
     call MakeStringMap as intervalsMap {input: keys=ground_truth_files, values=ground_truth_intervals}
     call MakeStringMap as lablesMap    {input: keys=ground_truth_files, values=truth_labels}
     call MakeStringMap as indexesMap   {input: keys=ground_truth_files, values=ground_truth_indexes}
+    call MakeStringMap as evalLabelsMap {input: keys=input_callset,     values = input_callset_labels}
```

Comment:
Stylistic comment: the spaces around `=` should be removed here, so that the values are `values=input_callset_labels` to be consistent with previous lines.